samplomatic/samplex/samplex.py

```diff
@@ -416,7 +416,7 @@ def sample(
             if key.startswith("measurement_flips"):
                 outputs[key][:] = 0
 
-        rng = default_rng(rng) if isinstance(rng, (int, SeedSequence)) else (rng or RNG)
+        rng = default_rng(rng) if isinstance(rng, (int, SeedSequence)) else (rng or RNG)  # noqa: UP038
```

Comment:
Since this won't affect any benchmark, I'd rather obey the rule everywhere, or ignore the rule at the root level, rather than on a per-line basis. For that reason I'm suggesting:

```suggestion
        rng = default_rng(rng) if isinstance(rng, int | SeedSequence) else (rng or RNG)
```

Reply:
Sounds good!